account_invoice_show_currency_rate/models/account_move.py

```diff
@@ -34,7 +34,10 @@ def _compute_currency_rate_amount(self):
             if item.state == "posted" and lines:
                 amount_currency_positive = sum(lines.mapped("amount_currency"))
                 total_debit = sum(item.line_ids.mapped("debit"))
-                item.currency_rate_amount = amount_currency_positive / total_debit
+                item.currency_rate_amount = item.currency_id.round(
+                    (amount_currency_positive / total_debit),
+                    item.currency_id.decimal_places,
```

Comment:
I don't think this is the syntax of the method. It already knows the decimal places.